src/Components/SuggestedInstanceTable/SuggestedInstanceTable.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {
+    TableVariant,
+    TableComposable,
+    Thead, Tr, Th, Tbody, Td
+} from '@patternfly/react-table';
+import {
+    Card,
+    CardBody,
+    Pagination,
+    PaginationVariant
+} from '@patternfly/react-core';
+import { PrimaryToolbar } from '@redhat-cloud-services/frontend-components/PrimaryToolbar';
+import { PAGE, PER_PAGE } from '../../constants';
+
+const COLUMNS = [
```

Comment:
Nitpick:

```suggestion
const SUGG_INSTANCE_TYPES_TABLE_COLUMNS = [
```
 
 I would prefer more specific name than generic term to avoid conflicts.